app/components/App.jsx

```diff
@@ -3,10 +3,11 @@ import Counter from './Counter';
 
 export default class App extends React.Component {
   render() {
-    return (
+    const widget = (
```

Comment:
This useless variable wrapping is done so you can still keep class based components as examples, otherwise the linter recommends pure functional components(and it is right, but for the sake of examples, I did this)